flocker/acceptance/endtoend/test_dockerplugin.py

```diff
@@ -146,6 +152,62 @@ def _create_volume(self, client, name, driver_opts):
         self.addCleanup(client.remove_volume, name)
         return result
 
+    def _test_sized_vol_container(self, cluster, node):
+        """
+        Create a volume with a size, then create a
+        container that can use that volume and run lsbk to test
+        its size. An http server inside the container uses lsblk to
+        return the size of the volume when path is not "/is_http"
```

Comment:
Nit: please end sentence with '.' and use a blank line between description and `param` directives.